### DISCUSSION TITLE
Is the styling of the documentation only for me strange?

### DISCUSSION THREAD

**Discussion by rmplbv** [vitaly-t/pg-promise#880] [General]
Since a while I got a vertical scrollbar overlapping with the content. Is it only me?
Also - prob. because of dark mode at github - the menu is black, the rest is white?

![image](https://github.com/vitaly-t/pg-promise/assets/44094912/b3996903-7f3b-4121-90a2-14c0aad8fc97)

**Comment by vitaly-t** [comment]
I only tested it for Chrome + Safari, never saw any issue with the scrolling. You are probably using a different browser. And if that's the case - feel free submitting a PR with the fix.

Color-wise, it is the same everywhere. It is quite an old documentation setup.

**Reply by rmplbv** [reply]
I'm on the latest version of Chrome. Does the left menu fit on one screen at your setup? It disappears if reducing font size so much, that no scrollbar is necessary. If I'll have some spare time i'll try to fix and submit PR.

Anyway, thanks a lot for pg-promise - it's awesome!
